### PR TITLE
Fix #653, finesse selected tab designs

### LINKED ISSUE #653: Tabs - selected state
The selected state for tabs is currently different than what was mocked. We should clean this up before we launch.

### PR DESCRIPTION
**Pull request by helenvholmes**
Naming things in ```variables.css``` is hard and I dislike it

### CODE REVIEW

File: public/js/components/SourceTabs.css [hunk 1]
```diff
@@ -10,17 +10,19 @@
 }
 
 .source-tab {
-  border: 1px solid var(--theme-gray);
+  background-color: var(--theme-toolbar-background-alt);
+  color: var(--theme-faded-tab-background);
```
Comment (jasonLaster) [inline public/js/components/SourceTabs.css]: should this be `-theme-faded-tab-color`?

File: public/js/lib/themes/variables.css [hunk 2]
```diff
@@ -20,7 +20,9 @@
   --theme-contrast-background: #e6b064;
 
   --theme-tab-toolbar-background: #fcfcfc;
+  --theme-faded-tab-background: #7E7E7E;
   --theme-toolbar-background: #fcfcfc;
+  --theme-toolbar-background-alt: #f5f5f5;
```
Comment (jasonLaster) [inline public/js/lib/themes/variables.css]: would it be possible to commit these changes to gecko as well so the changes are consistent.

**Comment by jasonLaster** [comment]
@helenvholmes i agree, CSS variable names are hard...

I'd like to revisit some of our CSS strategy once we land in firefox. I think now that we can safely plan on being an independent project in github, we can discuss some CSS strategies for the debugger as long as it would make sense for other devtool tools in the future as well.

File: public/js/components/SourceTabs.css (hunk 1 above)
Comment (helenvholmes) [inline public/js/components/SourceTabs.css]: Meaning as a variable name?

**Comment by jasonLaster** [comment]
👍

**Comment by wldcordeiro** [comment]
@jasonLaster I'll throw an oddball suggestion into the mix. https://github.com/cssinjs/jss 😄

**Comment by jlongster** [comment]
We've talked about a CSS in JS solution before but I want more buy-in from the team first, especially @helenvholmes. I think there are a lot of benefits to it but I want others to know how to work with it too.

**Comment by jlongster** [comment]
The linting errors were my fault. I'll merge this in as it seems good to me too!

**Pull request merged**